app/models/user.rb

```diff
@@ -1,10 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Hi @GordonOtieno, based on the requirements of this project, your app should request the user to confirm his/her email hence `:confirmable` should be enabled in your user model. I wasn't ask to confirm my email. Let's kindly fix :wrench: it